.claude/audit/scripts/diff.py

```
@@ -0,0 +1,110 @@
+        # by-hand editorial gate (see ../README.md "Triage"), never by this script.
+        rec = {"type": extern, "label": label, "n_distinct": len(distinct),
+               "mentioned_in_skill": mentioned, "missing_from_skill": missing}
+        if comp_mentioned and missing:
+            tier1.append(rec)
+        elif not comp_mentioned:
+            tier2.append(rec)
+
+    tier1.sort(key=lambda r: -len(r["missing_from_skill"]))
+    tier2.sort(key=lambda r: -r["n_distinct"])
+    os.makedirs(os.path.dirname(args.out) or ".", exist_ok=True)
+    json.dump({"tier1": tier1, "tier2": tier2}, open(args.out, "w", encoding="utf-8"), indent=1, ensure_ascii=False)
```

Ruff (0.15.14)

[warning] Use a context manager for opening files

(SIM115)